dm/pkg/log/log.go

```diff
@@ -194,3 +195,34 @@ func WrapStringerField(message string, object fmt.Stringer) zap.Field {
 func WithCtx(ctx context.Context) Logger {
 	return Logger{appLogger.With(getZapFieldsFromCtx(ctx)...)}
 }
+
+var (
+	enabledRedactLog atomic.Value
+)
```

Comment:
From `make fmt; git diff`:

```suggestion
var enabledRedactLog atomic.Value
```